core/components/header/index.tsx

```diff
@@ -52,60 +58,62 @@ export const Header = async ({ cart, data }: Props) => {
 
   return (
-    <ComponentsHeader links={categoryTree} logo={logo}>
-      <QuickSearch logo={logo} />
-
-      {customerId ? (
-        <Dropdown
-          items={[
-            { path: '/account', name: 'My account' },
-            { path: '/account/orders', name: 'Orders' },
-            { path: '/account/messages', name: 'Messages' },
-            { path: '/account/addresses', name: 'Addresses' },
-            { path: '/account/wishlists', name: 'Wish lists' },
-            { path: '/account/recently-viewed', name: 'Recently viewed' },
-            { path: '/account/settings', name: 'Account settings' },
-            { action: logout, name: 'Log out' },
-          ]}
-          trigger={
-            <Button
-              aria-label="Account"
-              className="p-3 text-black hover:bg-transparent hover:text-primary"
-              variant="subtle"
-            >
-              <User aria-hidden="true" />
-            </Button>
-          }
-        />
-      ) : (
-        <Link aria-label="Login" className="block p-3" href="/login">
-          <User />
-        </Link>
-      )}
-      <p role="status">
-        <Suspense
-          fallback={
-            <CartLink>
-              <ShoppingCart aria-label="cart" />
-            </CartLink>
-          }
-        >
-          {cart}
-        </Suspense>
-      </p>
-      <NextIntlClientProvider locale={locale} messages={{ Header: messages.Header ?? {} }}>
-        <LocaleSwitcher />
-      </NextIntlClientProvider>
-    </ComponentsHeader>
+    <ComponentsHeader
+      account={
+        customerId ? (
+          <Dropdown
+            items={[
+              { href: '/account', label: 'My account' },
+              { href: '/account/orders', label: 'Orders' },
+              { href: '/account/messages', label: 'Messages' },
+              { href: '/account/addresses', label: 'Addresses' },
+              { href: '/account/wishlists', label: 'Wish lists' },
+              { href: '/account/recently-viewed', label: 'Recently viewed' },
+              { href: '/account/settings', label: 'Account settings' },
+              { action: logout, name: 'Log out' },
+            ]}
+            trigger={
+              <Button
+                aria-label="Account"
+                className="p-3 text-black hover:bg-transparent hover:text-primary"
+                variant="subtle"
+              >
+                <User aria-hidden="true" />
+              </Button>
+            }
+          />
+        ) : (
+          <Link aria-label="Login" className="block p-3" href="/login">
+            <User />
+          </Link>
+        )
+      }
+      activeLocale={locale}
+      cart={
+        <p role="status">
+          <Suspense
+            fallback={
+              <CartLink>
+                <ShoppingCart aria-label="cart" />
+              </CartLink>
+            }
+          >
+            {cart}
+          </Suspense>
+        </p>
+      }
```

Comment:
There's a chance you probably already discussed this with Alan - are we going with `ReactNode` for `cart` prop instead of `cartHref` and `cartCount`?

Reply:
Yes, for now, we'll keep as is in Catalyst.